bitnami/grafana-operator/Chart.yaml

```diff
@@ -30,4 +30,4 @@ maintainers:
 name: grafana-operator
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
-version: 4.1.0
+version: 4.1.1
```

Comment:
As you are adding a new parameter, we need to bump the version to a new minor one.

```suggestion
version: 4.2.0
```